src/mdtest-main.c

```diff
@@ -2,12 +2,9 @@
 #include "aiori.h"
 
 int main(int argc, char **argv) {
-    aiori_initialize(NULL);
```

Comment:
Great patch. Could you clarify why you removed aiori init and finalize from main?

Reply:
if i remember right, the init and finalize calls, at least for the DFS driver, require input from the user (pool, container uuids) and those are not done until we parse the command line options in mdtest_run().
please let me know if you have a better suggestion and i can update.